low-level/framework/base/sspl_conf_adopt.py

```diff
@@ -44,3 +44,4 @@ def update_sub_section_diff(self):
     conf_diff.update_sub_section_diff()
     with open('/tmp/sspl_tmp.conf', 'w') as configfile:
         conf_diff.conf_file2_dict.write(configfile, space_around_delimiters=False)
+        
```

Comment:
avoid blanks, spaces. so file doesnt unnecessarily show in already big patch

Reply:
Done